crates/apollo-compiler/src/database/hir.rs

```diff
@@ -317,18 +318,6 @@ impl std::fmt::Display for OperationType {
     }
 }
 
-impl From<OperationType> for String {
-    fn from(op_type: OperationType) -> Self {
-        if op_type.is_subscription() {
-            "Subscription".to_string()
-        } else if op_type.is_mutation() {
-            "Mutation".to_string()
-        } else {
-            "Query".to_string()
-        }
-    }
-}
-
```

Comment:
i think removing this is technically a breaking change eh?

Reply:
Yes you can no longer do `op_type.into()`, you have to do `op_type.to_string()` instead. So very minor but breaking nonetheless